locales/en.default.json

```diff
@@ -51,7 +51,8 @@
     }
   },
   "date_formats": {
-    "month_year": "%B %Y"
+    "month_year": "%B %Y",
+    "month_day_year": "%B %d, %Y"
```

Comment:
From my understanding, some of the [`format` filter shorthand](https://shopify.dev/api/liquid/filters/html-filters#format) are already taking language translation in consideration so we wouldn't need this.